core/src/view/view.cpp

```diff
@@ -69,6 +69,7 @@ void View::setSize(int _width, int _height) {
     m_aspect = (float)m_vpWidth/ (float)m_vpHeight;
     m_dirtyMatrices = true;
     m_dirtyTiles = true;
+    m_dirtyConstraint = true;
```

Comment:
Looks like this is never set to false?

Reply:
The calculated value also depends on `pixelScale` so logically we'd need to reset the flag in that setter too.